app/models/protocol.rb

```diff
@@ -129,6 +129,10 @@ def is_study?
   def virgin_project?
     selected_for_epic == nil
   end
+  # returns true if selected_for_epic is nil in the database
+  def selected_for_epic_is_nil_in_db?
```

Comment:
What is this?  Seriously, is this a method.  What about just "selected_for_epic.nil?"
You already have the object.  Why are you re-querying?